index.html

```diff
@@ -5,9 +5,24 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <link rel="stylesheet" href="styles.css" />
     <title>Currency Converter</title>
+     <script src="https://kit.fontawesome.com/0f4d71be75.js" crossorigin="anonymous"></script>
+     
   </head>
   <body>
     <h1>Currency Converter App</h1>
+    <div class="search-form">
+      <h3>CURRENCY SEARCH</h3>
+      <form id="searchForm">
+      <input type="text" id="searchFrom" name="SearchFrom" placeholder="Base Currency (e.g. USD)" Required>
```

Comment:
Both inputs can not be required. The brief says `a function to find a specific currency rate, searching by the currency from OR to`. Meaning I can search the following examples:
- from EUR to DKK
- from USD (`to` is empty)
- to SKK (`from` is empty)

Reply:
Thank you for your feedback @vdaguenet. Could you please throw more light on this?
Do I just need to remove "Required from the searchFrom input field? or the search form you have just on input field instead of 2 fields?

Reply:
For the form, you only need to remove the `required` attributes but you keep both fields.
Then the work will be on your search function. Here is the expected result in pseudo code:
```js
function searchRate(baseCurrency, toCurrency) {
  // performs searching and returns a currency rate object
}

searchRate('EUR', 'DKK'); // returns { base: 'EUR', rates: { 'DKK': 123 } }
searchRate(null, 'DKK'); // returns [{ base: 'EUR', rates: { 'DKK': 123 } }, { base: 'USD', rates: { 'DKK': 234 } }]
searchRate('USD', null); //  return { base: 'USD', rates: { 'DKK': 234 } }
```